File: BW8AGU/VB/Ch08/BW8AGU.Samples.VB/BW8AGU.Samples.VB/DataModel/SimpleAppointment.vb
Public Class SimpleAppointment
    Public Property Id() As Integer
        Get
            Return m_Id
        End Get
        Set(value As Integer)
            m_Id = value
        End Set
    End Property
    Private m_Id As Integer
    Public Property Subject() As String
        Get
            Return m_Subject
        End Get
        Set(value As String)
            m_Subject = Value
        End Set
    End Property
    Private m_Subject As String
    Public Property Location() As String
        Get
            Return m_Location
        End Get
        Set(value As String)
            m_Location = Value
        End Set
    End Property
    Private m_Location As String
    Public Property StartDate() As DateTime
        Get
            Return m_StartDate
        End Get
        Set(value As DateTime)
            m_StartDate = Value
        End Set
    End Property
    Private m_StartDate As DateTime
    Public Property EndDate() As DateTime
        Get
            Return m_EndDate
        End Get
        Set(value As DateTime)
            m_EndDate = Value
        End Set
    End Property
    Private m_EndDate As DateTime
End Class

File: BW8AGU/C#/Ch08/BW8AGU.Samples.CSharp/DataModel/SimpleAppointment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SQLite;

namespace BW8AGU.Samples.CSharp.DataModel
{
    public class SimpleAppointment
    {
        [SQLite.PrimaryKey]
        [SQLite.AutoIncrement]
        public int Id { get; set; }
        public string Subject { get; set; }
        public string Location { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
    }
}
